configuration/addresshierarchy/addresshierarchy.csv

```diff
@@ -1,43 +1,19 @@
-Dessalines%01
-Gonaïves%02
-Gros-Morne%03
-Marmelade%04
-Saint-Marc%05
-Cerca-la-Source%06
-Hinche%07
-Lascahobas%08
-Mirebalais%09
-Anse d'Hainault%10
-Corail%11
-Jérémie%12
-Anse-à-Veau%13
-Baradères%14
-Miragoâne%15
-Acul-du-Nord%16
-Borgne%17
-Cap-Haïtien%18
-Grande-Rivière-du-Nord%19
-Limbé%20
-Plaisance%21
-Saint-Raphaël%22
-Fort-Liberté%23
-Ouanaminthe%24
-Trou-du-Nord%25
-Vallières%26
-Môle-Saint-Nicholas%27
-Port-de-Paix%28
-Saint-Louis-du-Nord%29
-Arcahaie%30
-Croix-des-Bouquets%31
-Gonâve (island)%32
-Léogâne%33
-Port-au-Prince%34
-Bainet%35
-Belle-Anse%36
-Jacmel%37
-Aquin%38
-Les Cayes%39
-Chardonnières%40
-Côteaux%41
-Port-Salut%42
-Port-Salut%43
\ No newline at end of file
+Santo Domingo|Santo Domingo Norte|Neighbborhood1|Street1

```

Comment:
Seems like a typo has sneaked in: `Neighbborhood1`

Reply:
Oops ...let me fix that.